### DISCUSSION TITLE
Any way to set a fallback symbols replacement?

### DISCUSSION THREAD

**Discussion by Hekzory** [JanDeDobbeleer/oh-my-posh#5033] [Troubleshoot]
Basically: 
I use four shells+terminals from time to time - zsh+wezterm, zsh+konsole, zsh+tty, powershell+windows terminal.

I want to have a single config for all of them, but currently most annoying blocker is not having icon symbols support in tty, so I fall back to having half of symbols replaced by placeholder symbol. 

The question is: how do I tell config to fallback to simple alternatives defined by me(for example, instead of fancy "❯" symbol use ">" if first is not present).

EDIT: I can probably achieve desired behaviour by using templates, could anyone suggest how to properly check if symbols are available or if i am in tty? Maybe get something from environment variables?

**Comment by JanDeDobbeleer** [comment]
@Hekzory `var` could be an option but they aren't template enabled right now. I agree that for this use-case templates are an option, albeit rather cumbersome. If I expand `var` to `vars`, with the same template enabled switch like we do for palettes, this could be a rather straightforward addition to the configuration.

**Reply by Hekzory** [reply]
Right now I went with this solution:

I determine if the terminal could use icons via zshrc, if it cannot, I create a new environment variable and set it to 1.

Right now my detection in zshrc is simple, if the amount of colors supported is lower than 256, then it is a low-capability terminal and I don't expect it to load icons. This solution works in my cases, but pretty sure it is not 100% foolproof.

Then in templates I check if there is such env variable present, and display an icon or its alternative.

However, would it be possible for oh-my-posh to expose some terminal parameters like the amount of colors supported via readily-available template variables or something like that? Especially if there is such data which is already retrieved by posh anyway, since it would eliminate the need of getting such info through external scripts, which is a bit messy, I think.

**Reply by JanDeDobbeleer** [reply]
@Hekzory that information is available using the environment variable `COLORTERM` which is set to `truecolor` in those cases
